### PR TITLE
Add new info command

### PR DESCRIPTION
**Pull request by swissspidy**
**Description**
Adds a new `wp traduttore info` command that prints various details about the environment, similar to `wp cli info`.

See #67.

**How has this been tested?**

Ran the command locally with the following output:

```bash
$ wp @translate-dev traduttore info
Traduttore version:	3.0.0-alpha
WordPress version:	4.9.8
GlotPress version:  2.3.1
WP-CLI version:		2.0.1
WP-CLI binary path:	/usr/local/bin/wp
Git binary path:	/usr/bin/git
Mercurial binary path:	(not found)
Subversion binary path:	(not found)
Cache directory:	/var/www/network.required.com/wordpress/content/traduttore
```

**Types of changes**

New feature (non-breaking change which adds functionality)

**Checklist:**
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code has proper inline documentation.

### CODE REVIEW

File: inc/CLI/InfoCommand.php [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+ *     Traduttore version:     3.0.0-alpha
+ *     WordPress version:      4.9.8
+ *     WP-CLI version:         2.0.1
+ *     WP-CLI binary path:     /usr/local/bin/wp
+ *     Git binary path:        /usr/bin/git
+ *     Mercurial binary path:  (not found)
+ *     Subversion binary path: (not found)
+ *     Cache directory:        /var/www/network.required.com/wordpress/content/traduttore
+ *
+ * @since 2.0.0
```
Comment (ocean90) [inline inc/CLI/InfoCommand.php]:
```suggestion
 * @since 3.0.0
```

File: inc/CLI/InfoCommand.php [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+ *     # Display various data about the Traduttore environment
+ *     $ wp traduttore info
+ *     Traduttore version:     3.0.0-alpha
+ *     WordPress version:      4.9.8
+ *     WP-CLI version:         2.0.1
+ *     WP-CLI binary path:     /usr/local/bin/wp
+ *     Git binary path:        /usr/bin/git
+ *     Mercurial binary path:  (not found)
+ *     Subversion binary path: (not found)
+ *     Cache directory:        /var/www/network.required.com/wordpress/content/traduttore
```
Comment (grappler) [inline inc/CLI/InfoCommand.php]:
```suggestion
 *     Cache directory:        /var/www/network.required.com/wp-content/traduttore
```
I would change it to a standard folder structure

File: inc/CLI/InfoCommand.php [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+ *   - list
+ *   - json
+ * ---
+ *
+ * ## EXAMPLES
+ *
+ *     # Display various data about the Traduttore environment
+ *     $ wp traduttore info
+ *     Traduttore version:     3.0.0-alpha
+ *     WordPress version:      4.9.8
```
Comment (grappler) [inline inc/CLI/InfoCommand.php]: We could add GlotPress version too.

**Comment by swissspidy** [comment]
> Would it make sense to add some information about the cron job, the WebHook URL or secrets?

Cron job: I can't think of any valuable information here. Do you have an example? Otherwise there's `wp cron`.

Webhook URL: not very useful IMO but can be added

Secrets: I'm not sure this is something that should be exposed via this command... Plus, secrets could be defined on a per-project basis. So perhaps it's more something for a `wp traduttore debug <project>` command (#67) if anything.

**Comment by swissspidy** [comment]
Looks like I cannot merge this without review approval from @ocean90. Unless I perhaps dismiss the review 🤔

**Pull request merged**